src/FSharp.Data.GraphQL.Client/BaseTypes.fs

```diff
@@ -112,16 +111,22 @@ type RecordBase (name : string, properties : RecordProperty seq) =
     /// Gets a list of this provided record properties.
     member __.GetProperties() = properties
     
-    /// Produces a dictionary containing all the properties and names of this provided record type.
+    /// Produces a dictionary containing all the properties of this provided record type.
     member x.ToDictionary() =
         let rec mapper (v : obj) =
```

Comment:
Minor nitpick, could we give `mapper` a more descriptive name?

Reply:
Done.